src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,20 +56,39 @@
   // private final XboxController m_controller = new XboxController(0);
   private Joystick joystick1;
   private Joystick joystick2;
+  private Joystick joystick3;
+
+    // joystick1 buttons
+  private JoystickButton driverIntakeButton;
+  private JoystickButton driverFeedButton;
+  private JoystickButton driverEjectButton;
+  private JoystickButton driverStartShootin;
 
   // joystick2 buttons
   private Button resetGyro;
   private JoystickButton startShootin;
   private JoystickButton stopShootin;
```

Comment:
add commented // @todos on these to remove later

Reply:
I just removed them